### PR TITLE
qrtool: 0.8.4 -> 0.8.5

### PR DESCRIPTION
**Pull request by philiptaron**
## Description of changes

https://github.com/sorairolake/qrtool/compare/v0.8.4...v0.8.5

## Things done

Updated to the [0.8.5 tag](https://github.com/sorairolake/qrtool/tree/v0.8.5). I also added in the man pages and shell completions from the project, as I've just now learned how to do that.

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by atalii — CHANGES_REQUESTED** [review]
I'm not on my NixOS machine right now, so I'll test functionality &c later today. Aside from the let-in, though, this all looks good to me!

File: pkgs/by-name/qr/qrtool/package.nix [hunk 1]
```diff
@@ -2,27 +2,42 @@
...
-  pname = "qrtool";
-  version = "0.8.4";
+, asciidoctor
+, installShellFiles
+}: let
+  name = "qrtool";
+  version = "0.8.5";
+in rustPlatform.buildRustPackage {
+  pname = name;
+  inherit version;
```
[the remark this answers is not in the document]
Comment (philiptaron) [reply pkgs/by-name/qr/qrtool/package.nix:12]: It won't, no. I thought it was a strictly less powerful version of `rec` (not liable to introduce any other names into the scope other than what I intended) so I picked it. I can revert.

The `finalAttrs` thing is neat but not supported by `buildRustPackage`, as you point out.

[the remark this answers is not in the document]
Comment (atalii) [reply pkgs/by-name/qr/qrtool/package.nix:12]: I'd revert. `rec` is easier to read and used pretty consistently across buildRustPackage invocations.

[the remark this answers is not in the document]
Comment (philiptaron) [reply pkgs/by-name/qr/qrtool/package.nix]: Done.

**Review by SuperSandro2000 — APPROVED** [review]

**Review by atalii — APPROVED** [review]
lgtm. thanks!

**Comment by sorairolake** [comment]
[v0.9.0](https://github.com/sorairolake/qrtool/releases/tag/v0.9.0) has been released.

**Pull request merged**